projects/scion/toolkit/uuid/src/uuid.util.ts

```diff
@@ -9,26 +9,36 @@
  */
 
 /**
- * Allows generating pseudo-random identifiers.
+ * Fast UUID generator, RFC4122 version 4 compliant.
+ *
+ * Usage: UUID.randomUUID()
  */
-export class UUID {
+function* generateUUID(): IterableIterator<string> {
```

Comment:
Can we move the generator and const into the namespace as well?

```ts
/**
 * Fast UUID generator, RFC 4122 version 4 compliant.
 *
 * Usage:
 *
 * ```ts
 * UUID.randomUUID();
 * ```
 */
export namespace UUID {

  const uuidIterator = generateUUID();

  function* generateUUID(): IterableIterator<string> {
  ...
  }

  /**
   * Generates a UUID compliant with the RFC 4122 version 4 standard.
   */
  export function randomUUID(): string {
    return uuidIterator.next().value;
  }
}
```

Reply:
As I already said, there is no `namespace` anymore.